packages/cactus-common/src/test/typescript/unit/js-object-signer.test.ts

```diff
@@ -200,21 +191,17 @@ test("Test optional hash function", async (assert: Test) => {
   const outer2 = { outerProperty: "test", innerProperty: inner2 };
   const sign2 = jsObjectSigner.sign(outer2);
 
-  assert.equals(sign1.toString, sign2.toString);
-  assert.end();
+  expect(sign1.toString).toEqual(sign2.toString);
 });
 
-test("Test missing required constructor field", async (assert: Test) => {
+test("Test missing required constructor field", async () => {
   try {
     const pkey: unknown = undefined;
     const jsObjectSignerOptions: IJsObjectSignerOptions = {
       privateKey: pkey as Uint8Array,
     };
     new JsObjectSigner(jsObjectSignerOptions);
-  } catch (e) {
-    if (e instanceof Error) {
-      assert.equal(e.message, "JsObjectSigner#ctor options.privateKey falsy.");
-    }
+  } catch (e: any) {
```

Comment:
@awadhana Instead of `any` please qualify it as `unkown` and then you'll need the type narrowing if condition back too to avoid compilation errors.
Plus you could also add an else block that fails the test to fix the original issue [1] (which I assume you were fixing by removing the if condition entirely)

[1] => If the exception thrown ends up not being an instance of `Error` then the assertion will not happen at all, that was a problem with the original code, which is not related to the jestification but it's great that you fixed that part of it.

Reply:
@petermetz, Tony and I tried something different, hopefully the tests pass through :)

Reply:
@awadhana Perfect, thank you for making the change!